packages/core/admin/admin/src/content-manager/components/BlocksInput/utils/conversions.ts

```diff
@@ -89,4 +91,27 @@ const insertEmptyBlockAtLast = (editor: Editor) => {
   );
 };
 
-export { baseHandleConvert, getAttributesToClear, isLastBlockType, insertEmptyBlockAtLast };
+function useConversionModal() {
```

Comment:
Would it not be appropriate to have a hooks folder?

Reply:
we only have this one, I'm not sure it's worth it. I agree that it's weird to have react code here though. I'll try exporting from BlocksToolbar if that works for you